smart-cards/src/main/java/gov/va/api/health/smartcards/patient/PatientController.java

```diff
@@ -178,15 +187,20 @@ private static VerifiableCredential vc(MixedBundle bundle, List<CredentialType>
                     credentialTypes.stream().map(CredentialType::getUri))
                 .collect(toList()))
         .credentialSubject(
-            VerifiableCredential.CredentialSubject.builder().fhirBundle(bundle).build())
+            VerifiableCredential.CredentialSubject.builder()
+                .fhirVersion("4.0.1")
+                .fhirBundle(bundle)
+                .build())
         .build();
   }
 
   @PostMapping(value = "/{id}/$health-cards-issue")
   ResponseEntity<Parameters> healthCardsIssue(
       @PathVariable("id") String id,
       @Valid @RequestBody Parameters parameters,
-      @RequestHeader(name = "Authorization", required = false) String authorization) {
+      @RequestHeader(name = "Authorization", required = false) String authorization,
+      @RequestHeader(name = "x-vc-jws", required = false) String vcJws,
+      @RequestHeader(name = "x-vc-compress", required = false) String vcCompress) {
```

Comment:
Out of curiosity, does Spring support mapping request headers to `Boolean`?

Reply:
It might but I want it to default to true in the case the value can't parse to a boolean. So only the exact value of `false` would disable this functionality.